src/capi_frontend/server_settings.hpp

```diff
@@ -111,21 +118,25 @@ struct TextGenGraphSettingsImpl {
 };
 
 struct EmbeddingsGraphSettingsImpl {
+    EmbeddingsGraphSettingsImpl() :
+        pluginConfig{std::nullopt, std::nullopt, std::nullopt, 1} {}
     std::string modelPath = "./";
     std::string targetDevice = "CPU";
     std::string modelName = "";
-    uint32_t numStreams = 1;
     std::string normalize = "true";
     std::string truncate = "false";
     std::string pooling = "CLS";
+    PluginConfigSettingsImpl pluginConfig;
 };
 
 struct RerankGraphSettingsImpl {
+    RerankGraphSettingsImpl::RerankGraphSettingsImpl() :
```

Comment:
```suggestion
    RerankGraphSettingsImpl() :
```
- It fixes building on Linux